src/Morphir/JsonSchema/Backend.elm

```diff
@@ -46,7 +46,7 @@ type alias QualifiedName =
 
 
 type alias Error =
-    String
+    List String
```

Comment:
This is a bit confusing because `Error` is singular but it represents a collection of errors. I think having two types would make things cleaner and easier to scale:

```elm
type alias Errors =
    List Error

type alias Error =
    String
```

Reply:
This is clear. I taught of it too but wanted to keep simple!